### ISSUE TITLE
Support for Node 12

### ISSUE THREAD

**Issue by beevelop** [lukechilds/keyv-sqlite#51]
The `sqlite3` package referenced in `v2.0.0` of this package wasn't compatible with Node 12. Since `v4.0.7` it should be compatible: https://github.com/mapbox/node-sqlite3/issues/1151

The necessary changes have already been done in the current `master` branch, but not yet released. Any way to set a new tag and release a new version to npm?

**Comment by beevelop** [comment]
For all those looking for a quick fix: Just put the following resolution into you `package.json`:
```json
{
  "resolutions": {
    "@keyv/sqlite/**/sqlite3": "4.0.9"
  }
}
```

**Comment by lukechilds** [comment]
Published `@keyv/sqlite@2.0.1` with latest dep updates in master. Apologies for delay, haven't had much time for OSS recently.

Thanks for notifying me @beevelop 🙌

**Comment by beevelop** [comment]
You are awesome @lukechilds 🍻
